include/lo2s/monitor/main_monitor.hpp

```diff
@@ -82,6 +84,9 @@ class MainMonitor
 #ifdef HAVE_SENSORS
     std::unique_ptr<metric::sensors::Recorder> sensors_recorder_;
 #endif
+#ifdef HAVE_VEOSINFO
+    std::vector<std::unique_ptr<NecMonitorMain>> nec_monitors_;
```

Comment:
Why unique_ptr in vector?

Reply:
For the same reason we have tracepoint_monitors_ in std::vector<std::unique_ptr>. NecMonitorMain's have no default constructor